compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposeLayer.desktop.kt

```diff
@@ -74,21 +72,32 @@ import java.awt.event.WindowEvent
 import java.awt.event.WindowFocusListener
 import java.awt.im.InputMethodRequests
 import javax.accessibility.Accessible
-import javax.accessibility.AccessibleContext
 import javax.swing.SwingUtilities
 import kotlin.coroutines.AbstractCoroutineContextElement
 import kotlin.coroutines.CoroutineContext
-import java.awt.Cursor
-import java.awt.event.FocusListener
+import kotlinx.coroutines.CoroutineExceptionHandler
+import org.jetbrains.skia.Canvas
+import org.jetbrains.skiko.MainUIDispatcher
+import org.jetbrains.skiko.SkiaLayer
 import org.jetbrains.skiko.SkiaLayerAnalytics
-import org.jetbrains.skiko.hostOs
 import org.jetbrains.skiko.SkikoInput
+import org.jetbrains.skiko.SkikoView
+import org.jetbrains.skiko.hostOs
 
 internal class ComposeLayer(
     private val skiaLayerAnalytics: SkiaLayerAnalytics
 ) {
     private var isDisposed = false
 
+    internal val sceneAccessible = ComposeSceneAccessible(
```

Comment:
I will test only how it works (macOs, Windows) now later, and post the results here.

@m-sasha, could you do a full review?

Reply:
I checked it on MacOs and Windows, it works fine.